frame/fast-unstake/Cargo.toml

```diff
@@ -60,6 +60,7 @@ std = [
 runtime-benchmarks = [
 	"frame-benchmarking/runtime-benchmarks",
 	"frame-system/runtime-benchmarks",
-	"sp-staking/runtime-benchmarks"
+	"sp-staking/runtime-benchmarks",
+	"pallet-staking/runtime-benchmarks"
```

Comment:
why is this needed?

Reply:
try compiling `cargo test -p pallet-fast-unstake --features runtime-benchmarks`. Some of the method impls on `StkaingInterface` will be missing otherwise.